src/modules/Search/Search.js

```diff
@@ -57,7 +57,10 @@ export default class Search extends Component {
     noResultsDescription: PropTypes.string,
 
     /** Message to display when there are no results. */
-    noResultsMessage: PropTypes.string,
+    noResultsMessage: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.object,
+    ]),
```

Comment:
> PropTypes.node includes any renderable value: string, number, element, or an array (or fragment) of any of these. 

Didn't know about it. Thanks for useful information. Made the necessary changes.